docs/dependencies.md

```diff
@@ -57,6 +57,7 @@ to catalog our packages and provide guidance to a developer who wants to test an
 | [@wordpress/scripts](https://www.npmjs.com/package/@wordpress/scripts) | Collection of reusable scripts tailored for WordPress development. | JS tests and E2E pipeline works, which uses `wp-scripts` | `>=20` requires `jest >=27`, we're currently at `v26.x` so updated to  `v19.2.3`  |
 | [@woocommerce/explat](https://www.npmjs.com/package/@woocommerce/explat/) | Component and utility functions that can be used to run A/B Tests in WooCommerce dashboard and reports pages. | JS unit tests are passing | After update ensure types are still correct for `ExperimentProps`, or use a DefinitelyTyped package if available.  |
 | [@woocommerce/experimental](https://www.npmjs.com/package/@wordpress/plugins) | A collection of component imports and exports that are aliases for components transitioning from experimental to non-experimental.| JS unit tests are passing| Needed to install `@types/react-transition-group` as JS linting could not find a declaration file for module `react-transition-group/CSSTransition`. |
+| [@wordpress/dom-ready](https://www.npmjs.com/package/@wordpress/dom-ready) | Execute callback after the DOM is loaded.| JS unit tests are passing| Downgraded to match the minimum supported version of WordPress. |
```

Comment:
> Downgraded to match the minimum supported version of WordPress.

I don't think it's worth mentioning in this case
I see the "Notes" section as a way to point out some specificities related to the package which could be useful for the next person taking care of the update